### PR TITLE
upgrade to terraform 0.12

### PR DESCRIPTION
**Pull request by philandstuff**
Pretty dumb upgrade, this is mostly a no-op except for the fact there's a new ECS AMI so applying prom-ec2 will rebuild prometheus.

### CODE REVIEW

File: .terraform-version [hunk 1]
```diff
@@ -1 +1 @@
-0.11.13
+0.12.19
```
Comment (issyl0) [inline .terraform-version]: 0.12.20 is out now. 🙈

Comment (philandstuff) [reply .terraform-version]: don't care, wasn't when i started this 😄

**Review by szd55gds — CHANGES_REQUESTED** [review]
Hey @philandstuff , what a monster! My only concern is that in terraform/projects/infra-security-groups-production/main.tf, you remove comments which ID IP addresses to various GDS projects/services, I think this is a retrograde step because I can see folk in the near future asking "what on earth is that IP". Suggest we leave those comments in!

**Comment by philandstuff** [comment]
Good spot @szd55gds, I've pushed a commit to reinstate them

**Review by szd55gds — APPROVED** [review]
Looks magnificent to me :-)

**Pull request merged**